python/cugraph/community/leiden_wrapper.pyx

```diff
@@ -69,25 +72,23 @@ def leiden(input_graph, max_iter, resolution):
                                                   <float*>c_weights, num_verts, num_edges)
 
         graph_float.get_vertex_identifiers(<int*>c_identifier)
-        c_leiden(graph_float,
-                  final_modularity_float,
-                  num_level,
-                  <int*> c_partition,
-                  <int> max_iter,
-                  <float> resolution)
+        num_level, final_modularity_float = c_leiden(handle_ptr.get()[0],
```

Comment:
Nice, I didn't realize the pair could be unpacked like this. I was defining a cdef pair and accessing with `.first`, etc.  I'll need to change my WIP PR for consistency.